go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/konpyutaika/nifikop
 
-go 1.18
+go 1.20
```

Comment:
Fyi, a new 1.20.3 golang security release comes out on 2023-04-04. I don't know what's in it, but we might want to wait to build/merge until tomorrow so we can use it.

Reply:
@cannonpalms i see that, thanks for working on this